package.json

```diff
@@ -22,8 +22,7 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.10.0",
-    "ember-modifier-manager-polyfill": "^1.1.0"
```

Comment:
Should come back when we bring back the other ember-source versions in ember-try scenarios